<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>674</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="listWidget_outPath">
       <property name="minimumSize">
        <size>
         <width>221</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>411</width>
         <height>41</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="listWidget_ver">
       <property name="minimumSize">
        <size>
         <width>221</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>411</width>
         <height>41</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_chooseFile">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
